README.md

```diff
@@ -16,9 +18,45 @@ Loggly broadcasting for Good.
 - `good@6.x.x` is compatible with version 1.x.x and 2.x.x.
 - For older versions of `good`, use [v0.1.4](https://github.com/continuationlabs/good-loggly/releases/tag/v0.1.4).
 
-## Usage
+# 7.x.x Usage
 
-`good-loggly` is a [good](https://github.com/hapijs/good) reporter implementation to write [hapi](http://hapijs.com/) server events to Loggly.
+Example integration:
+
+```javascript
+var Hapi = require('hapi');
+var Good = require('good');
+
+var server = new Hapi.Server();
+var options = {
+    reporters: {
+        loggly: [{
+            module: 'good-squeeze',
+            name: 'Squeeze',
+            args: [{log: '*', request: '*', error: '*', response: '*'}]
+        }, {
+            module: 'good-loggly',
+            args: [{
+                token: 'YOUR LOGGLY TOKEN',
+                subdomain: 'YOUR LOGGLY SUBDOMAIN',
+                name: 'myapp',
+                hostname: 'myapp.example.com',
+                tags: ['global', 'tags', 'for', 'all', 'requests']
+            }]
+        }]
+    }
+};
+
+server.register({
+    register: Good,
+    options: options
+}, function (err) {
+    if (err) {
+        return console.error(err);
+    }
+});
+```
+
+## Pre 6.x.x Usage
```

Comment:
Can you remove the 6.x.x implementation? If someone wants the old README, they can just change the tag version on GitHub.